cmake/Modules/ObsHelpers_Windows.cmake

```diff
@@ -0,0 +1,274 @@
+
+    target_link_options(obs-browser PRIVATE "LINKER:/IGNORE:4099")
+
+    target_link_options(obs-browser-page PRIVATE "LINKER:/IGNORE:4099"
+                        "LINKER:/SUBSYSTEM:WINDOWS")
+  endif()
+
+  _setup_obs_app(${ARGV})
+
+  if(MSVC)
+    include(CopyMSVCBins)
+  endif()
+endfunction()
+
+# Helper function to do additional setup for browser source plugin
+function(setup_target_browser target)
+  _setup_target_browser(${ARGV})
+
+  if(DEFINED ENV{obsInstallerTempDir})
+    add_custom_command(
+      TARGET ${target}
+      POST_BUILD
+      COMMAND "${CMAKE_COMMAND}" -E copy_directory "${CEF_ROOT_DIR}/Resources/"
+              "$ENV{obsInstallerTempDir}/${OBS_PLUGIN_DESTINATION}"
+      COMMAND "${CMAKE_COMMAND}" -E copy_directory "${CEF_ROOT_DIR}/Release/"
+              "$ENV{obsInstallerTempDir}/${OBS_PLUGIN_DESTINATION}"
+      VERBATIM)
+  endif()
+endfunction()
+
+# Helper function to gather external libraries depended-on by libobs
+function(setup_libobs_target target)
+  set(_ADDITIONAL_FILES "${CMAKE_SOURCE_DIR}/additional_install_files")
+
+  if(DEFINED ENV{obsAdditionalInstallFiles})
+    set(_ADDITIONAL_FILES "$ENV{obsAdditionalInstallFiles}")
+  endif()
+
+  if(NOT INSTALLER_RUN)
+    list(APPEND _LIBOBS_FIXUPS "misc:." "data:${OBS_DATA_DESTINATION}"
+         "libs${_ARCH_SUFFIX}:${OBS_LIBRARY_DESTINATION}"
+         "exec${_ARCH_SUFFIX}:${OBS_EXECUTABLE_DESTINATION}")
+  else()
+    list(
+      APPEND
+      _LIBOBS_FIXUPS
+      "misc:."
+      "data:${OBS_DATA_DESTINATION}"
+      "libs32:${OBS_LIBRARY32_DESTINATION}"
+      "libs64:${OBS_LIBRARY64_DESTINATION}"
+      "exec32:${OBS_EXECUTABLE32_DESTINATION}"
+      "exec64:${OBS_EXECUTABLE64_DESTINATION}")
+  endif()
+
+  foreach(_FIXUP IN LISTS _LIBOBS_FIXUPS)
+    string(REPLACE ":" ";" _FIXUP ${_FIXUP})
+    list(GET _FIXUP 0 _SOURCE)
+    list(GET _FIXUP 1 _DESTINATION)
+
+    install(
+      DIRECTORY "${_ADDITIONAL_FILES}/${_SOURCE}/"
+      DESTINATION "${_DESTINATION}"
+      USE_SOURCE_PERMISSIONS
+      PATTERN ".gitignore" EXCLUDE)
+
+    add_custom_command(
+      TARGET ${target}
+      POST_BUILD
+      COMMAND
+        "${CMAKE_COMMAND}" -E copy_directory "${_ADDITIONAL_FILES}/${_SOURCE}/"
+        "${CMAKE_BINARY_DIR}/rundir/$<CONFIG>/${_DESTINATION}"
+      VERBATIM)
+
+    if(_SOURCE MATCHES "(libs|exec)(32|64)?")
+      install(
+        DIRECTORY "${_ADDITIONAL_FILES}/${_SOURCE}$<IF:$<CONFIG:Debug>,d,r>/"
+        DESTINATION "${_DESTINATION}"
+        USE_SOURCE_PERMISSIONS
+        PATTERN ".gitignore" EXCLUDE)
+
+      add_custom_command(
+        TARGET ${target}
+        POST_BUILD
+        COMMAND
+          "${CMAKE_COMMAND}" -E copy_directory
+          "${_ADDITIONAL_FILES}/${_SOURCE}$<IF:$<CONFIG:Debug>,d,r>/"
+          "${CMAKE_BINARY_DIR}/rundir/$<CONFIG>/${_DESTINATION}"
+        VERBATIM)
+    endif()
+  endforeach()
+endfunction()
+
+# Helper function to copy PDB files if available (binary/library
+# targets only)
```

Comment:
This should fit on one line unless I'm mistaken.

Reply:
Fixed.